src/Illuminate/Database/Eloquent/Builder.php

```diff
@@ -332,7 +332,7 @@ public function value($column)
      *
      * @param  int  $count
      * @param  callable  $callback
-     * @return bool
+     * @return  bool
```

Comment:
don't worry about phpdoc cs - as long a it's valid phpdoc, taylor or I can clean it up after merge